pac4j-couch/src/main/java/org/pac4j/couch/profile/service/CouchProfileService.java

```diff
@@ -0,0 +1,181 @@
+package org.pac4j.couch.profile.service;
+
+import org.pac4j.couch.profile.CouchProfile;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.ektorp.CouchDbConnector;
+import org.ektorp.DocumentNotFoundException;
+import org.ektorp.ViewQuery;
+import org.ektorp.ViewResult;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.credentials.password.PasswordEncoder;
+import org.pac4j.core.profile.definition.CommonProfileDefinition;
+import org.pac4j.core.profile.service.AbstractProfileService;
+import org.pac4j.core.util.CommonHelper;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The MongoDB profile service.
+ *
+ * @author Elie Roux
+ * @since 2.0.0
+ */
+public class CouchProfileService extends AbstractProfileService<CouchProfile> {
+
+    private CouchDbConnector couchDbConnector;
+    
+    public static final String COUCH_ID = "_id";
+    public static ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
would it make sense to give the possibility to the user to change the `ObjectMapper` (and thus it shouln't be static and also settable…).